.github/workflows/tests.yml

```diff
@@ -41,3 +65,20 @@ jobs:
           ( github.event_name != 'pull_request' || github.event.pull_request.head.repo.full_name == github.repository )
         with:
           files: ./**/build/test-results/**/*.xml
+
+# Notify the channel about runs against develop or main that have failures, as PRs should have caught these first.
+  notify:
+    runs-on: ubuntu-latest
+    needs:
+      - unit-tests
+      - build-android-tests
+    if: ${{ (github.ref == 'refs/heads/develop' || github.ref == 'refs/heads/main' ) && failure() }}
```

Comment:
NB: failure() does not fire if the above tests is cancelled due to concurrency kicking in - this should only run once.